### PR TITLE
(fix) O3-3906:  ComposedModal usage missing scroll bar

### PR DESCRIPTION
**Pull request by IamMujuziMoses**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Currently, using of ComposedModal in the Bed management module is missing a scroll bar, this PR adds a Scrollbar to the dialog to make it easy for different screen sizes.

## Related Issue

https://issues.openmrs.org/browse/O3-3906

### CODE REVIEW

**Comment by IamMujuziMoses** [comment]
@chibongho @vasharma05 @usamaidrsk  could you please review this PR, your feedback will be appreciated.

**Review by chibongho — APPROVED** [review]
LGTM. Thanks!

File: packages/esm-bed-management-app/src/bed-tag/bed-tags-admin-form.component.tsx [hunk 1]
```diff
@@ -110,16 +110,16 @@ const BedTagsAdministrationForm: React.FC<BedTagAdministrationFormProps> = ({
...
-          </Button>
-        </ModalFooter>
-      </Form>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button onClick={() => onModalChange(false)} kind="secondary">
+          {getCoreTranslation('cancel', 'Cancel')}
+        </Button>
+        <Button disabled={!isDirty} onClick={handleSubmit(onSubmit, onError)}>
```
Comment (denniskigen) [inline packages/esm-bed-management-app/src/bed-tag/bed-tags-admin-form.component.tsx:119]: I don't think we need the submit handler wired up both here and in the `Form` tag. The handler in the `Form` tag should be sufficient.

File: packages/esm-bed-management-app/src/bed-type/bed-type-admin-form.component.tsx [hunk 2]
```diff
@@ -149,16 +149,16 @@ const BedTypeAdministrationForm: React.FC<BedAdministrationFormProps> = ({
...
-          </Button>
-        </ModalFooter>
-      </Form>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button onClick={() => onModalChange(false)} kind="secondary">
+          {getCoreTranslation('cancel', 'Cancel')}
+        </Button>
+        <Button disabled={!isDirty} onClick={handleSubmit(onSubmit, onError)}>
```
Comment (denniskigen) [inline packages/esm-bed-management-app/src/bed-type/bed-type-admin-form.component.tsx:158]: Same here. I imagine we only need this on line 85.

**Review by denniskigen — COMMENTED** [review]
Nice work! I've left some suggestions on removing a duplicate submit handler.

File: packages/esm-bed-management-app/src/bed-type/bed-type-admin-form.component.tsx (hunk 2 above)
Comment (IamMujuziMoses) [reply packages/esm-bed-management-app/src/bed-type/bed-type-admin-form.component.tsx:158]: Thank you for the feedback, realized it works if I leave this `handleSubmit` implementation here and remove the one on line 85.

**Review by denniskigen — APPROVED** [review]

**Pull request merged**